### DISCUSSION TITLE
Aqara pet feeder C1 internal clock

### DISCUSSION THREAD

**Discussion by MichaelMure** [Koenkk/zigbee2mqtt#16157] [Devices]
I have this device and setup a scheduled feeding routine for the little monster.

The issue is that the schedule happen 1h off in my case. It seems that the device follow the UTC time zone instead of the local one. This begs two questions:
- would it be possible to make the device or zigbee2mqtt follow the local timezone?
- how does the device even know about the time? How does it stays accurate?

Maybe @kirovilya would know?

**Comment by kirovilya** [comment]
Yes, there is a time problem with the feeder C1.
During the interview, the device asks the coordinator the current time and it returns it in UTC - this code is in zigbee-herdsman and is common to all devices.
I haven't yet figured out how to override the response to the time request for a specific device - since this happens during the interview process and the onEvent code in the converters is not used, advice @Koenkk is needed
It may be necessary to somehow force the device to poll the current time again, but I do not know a way.

**Comment by kirovilya** [comment]
Found one solution... https://github.com/Koenkk/zigbee-herdsman-converters/pull/5364
